sys/indent.sh

```diff
@@ -52,6 +53,12 @@ if [ "${IFILE}" = "-u" ]; then
 	IFILE="$1"
 fi
 
+if [ "${IFILE}" = "-c" ]; then
```

Comment:
Why the double quoting of `-c`?

Reply:
It is just strange/inconsistent that this is the only word without special characters that is quoted. That is what I was trying to communicate.

Reply:
Honestly I just copied what was done for the previous parameters. What do you suggest?

Reply:
I, personally, would not quote it; but whatever.